lib/react_test_utils.dart

```diff
@@ -0,0 +1,227 @@
+
+  static dragStart(JsObject element, [Map eventData = const{}]) =>
+      _Simulate.callMethod('dragStart', [element, eventData]);
+
+  static drop(JsObject element, [Map eventData = const{}]) =>
+      _Simulate.callMethod('drop', [element, eventData]);
+
+  static focus(JsObject element, [Map eventData = const{}]) =>
+      _Simulate.callMethod('focus', [element, eventData]);
+
+  static input(JsObject element, [Map eventData = const{}]) =>
+      _Simulate.callMethod('input', [element, eventData]);
+
+  static keyDown(JsObject element, [Map eventData = const{}]) =>
+      _Simulate.callMethod('keyDown', [element, eventData]);
+
+  static keyPress(JsObject element, [Map eventData = const{}]) =>
+      _Simulate.callMethod('keyPress', [element, eventData]);
+
+  static keyUp(JsObject element, [Map eventData = const{}]) =>
+      _Simulate.callMethod('keyUp', [element, eventData]);
+
+  static mouseDown(JsObject element, [Map eventData = const{}]) =>
+      _Simulate.callMethod('mouseDown', [element, eventData]);
+
+  static mouseMove(JsObject element, [Map eventData = const{}]) =>
+      _Simulate.callMethod('mouseMove', [element, eventData]);
+
+  static mouseOut(JsObject element, [Map eventData = const{}]) =>
+      _Simulate.callMethod('mouseOut', [element, eventData]);
+
+  static mouseOver(JsObject element, [Map eventData = const{}]) =>
+      _Simulate.callMethod('mouseOver', [element, eventData]);
+
+  static mouseUp(JsObject element, [Map eventData = const{}]) =>
+      _Simulate.callMethod('mouseUp', [element, eventData]);
+
+  static paste(JsObject element, [Map eventData = const{}]) =>
+      _Simulate.callMethod('paste', [element, eventData]);
+
+  static scroll(JsObject element, [Map eventData = const{}]) =>
+      _Simulate.callMethod('scroll', [element, eventData]);
+
+  static submit(JsObject element, [Map eventData = const{}]) =>
+      _Simulate.callMethod('submit', [element, eventData]);
+
+  static touchCancel(JsObject element, [Map eventData = const{}]) =>
+      _Simulate.callMethod('touchCancel', [element, eventData]);
+
+  static touchEnd(JsObject element, [Map eventData = const{}]) =>
+      _Simulate.callMethod('touchEnd', [element, eventData]);
+
+  static touchMove(JsObject element, [Map eventData = const{}]) =>
+      _Simulate.callMethod('touchMove', [element, eventData]);
+
+  static touchStart(JsObject element, [Map eventData = const{}]) =>
+      _Simulate.callMethod('touchStart', [element, eventData]);
+
+  static wheel(JsObject element, [Map eventData = const{}]) =>
+      _Simulate.callMethod('wheel', [element, eventData]);
+
+}
+
+
+/// Traverse all components in tree and accumulate all components where
+/// test(component) is true. This is not that useful on its own, but it's
+/// used as a primitive for other test utils
+///
+/// Included in Dart for completeness
+List findAllInRenderedTree(JsObject inst, JsFunction test) {
+  return _TestUtils.callMethod('findAllInRenderedTree', [inst, test]);
+}
```

Comment:
They don't call this method, they call the JS method this method calls, but they perform that call outside of Dart.